core/server/api_container/server/startosis_engine/kurtosis_instruction/request/request.go

```diff
@@ -207,6 +209,11 @@ func (builtin *RequestCapabilities) FillPersistableAttributes(builder *enclave_p
 	builder.SetType(RequestBuiltinName)
 }
 
+func (builtin *RequestCapabilities) UpdatePlan(plan *plan_yaml.PlanYaml) error {
+	logrus.Warn("REQUEST NOT IMPLEMENTED YET FOR UPDATING PLAN")
```

Comment:
Note that plan generation is not only for docker compose imports - it renders yaml's for all packages. 

The thing is currently, the frontend doesn't support an `+Add a Request node` as it wasn't scoped before @Dartoxian offboarded so if we fail early, plan yaml generation fails for any package that contains the `request` instruction (which is a lot)- so for now (especially since Picasso/Enclave Builder is experimental), opting to just ignore effect of `request` instructions but agreed it's not ideal.

Reply:
throwing an error would indicate to the end person that this isn't accurate; maybe we can highlight it somehow

Reply:
Probably, can highlight it on the frontend somehow with a warning but opting to log error for now and can change as needed